WebGoatCore/Views/StatusCode/StatusCodeView.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@model StatusCodeViewModel
+
+<div>
+    @if(Model != null)
```

Comment:
To me it looks that Model can't be null (we create it with non null model) and also most other views do not check if Model is null